### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lrapetti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# Wear
\ No newline at end of file
+# Wearables
+
+## Wearable device sources
+Documentaton for running wearable data source can be found at the following links:
+- [FTShoes](https://github.com/robotology/wearables/blob/master/doc/How-to-run-FTshoes.md)
```
Comment (traversaro) [inline README.md]: If you do not hardcode the absolute URL, you ensure that the link will work correctly on branches/forks
```suggestion
- [FTShoes](doc/How-to-run-FTshoes.md)
```

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
...
+- [XsensSuit](https://github.com/robotology/wearables/blob/master/doc/How-to-run-XsensSuit.md)
```
Comment (traversaro) [inline README.md]:
```suggestion
- [XsensSuit](doc/How-to-run-XsensSuit.md)
```
If you do not hardcode the absolute URL, you ensure that the link will work correctly on branches/forks

File: README.md [hunk 3]
```diff
@@ -1 +1,6 @@
-# Wear
\ No newline at end of file
+# Wearables
+
+## Wearable device sources
+Documentaton for running wearable data source can be found at the following links:
```
Comment (traversaro) [inline README.md]:
```suggestion
Documentation for running wearable data source can be found at the following links:
```

File: README.md (hunk 2 above)
Comment (lrapetti) [reply README.md]: Thanks, changed.

**Pull request merged**